Content.Shared/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -233,6 +233,13 @@ public bool TryAddSolution(EntityUid targetUid, Solution? targetSolution, Soluti
             return solutionsMgr.Solutions.TryGetValue(name, out solution);
         }
 
+
+        public Solution GetSolution(EntityUid uid, string solutionName)
+        {
+            var solutionManager = ComponentManager.GetComponent<SolutionContainerManagerComponent>(uid);
```

Comment:
Should be TryGet IMO because it's possible someone calls this and causes an exception.

We already dissuade from GetComponent<T> from being used unless you're 100% sure it won't cause an exception so 99% of the time it needs to be a TryGet